### PR TITLE
PROGRAMMES-6392 fix snippet HTML

### PR DESCRIPTION
**Pull request by Tasiobg**
fix snippet title tag and some other changes to build same HTML as the snippet project does

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/ExternalApi/Tupac/Domain/Record.php [hunk 1]
```diff
@@ -157,4 +157,16 @@ public function getFormattedDuration(): string
         }
         return '';
     }
+
+    public function getTitleTag(): string
```
Comment (Tasiobg) [inline src/ExternalApi/Tupac/Domain/Record.php]: Same text and conditions as the original snippet

File: src/ExternalApi/Tupac/Domain/Record.php [hunk 2]
```diff
@@ -157,4 +157,16 @@ public function getFormattedDuration(): string
...
+    {
+        if (empty($this->resource) || empty($this->format)) {
+            $titleTag = 'This snippet is currently unavailable';
```
Comment (votemike) [inline src/ExternalApi/Tupac/Domain/Record.php]: Should this be translated?

File: src/ExternalApi/Tupac/Domain/Record.php [hunk 3]
```diff
@@ -157,4 +157,16 @@ public function getFormattedDuration(): string
...
+        } else if (!empty($this->title)) {
+            $titleTag = 'Play ' . $this->title;
```
Comment (votemike) [inline src/ExternalApi/Tupac/Domain/Record.php]: Should this be translated?

File: src/ExternalApi/Tupac/Domain/Record.php [hunk 4]
```diff
@@ -157,4 +157,16 @@ public function getFormattedDuration(): string
...
+        } else {
+            $titleTag =  'Play audio';
```
Comment (votemike) [inline src/ExternalApi/Tupac/Domain/Record.php]: Should this be translated?

File: templates/snippet/snippet_record.html.twig [hunk 5]
```diff
@@ -1,19 +1,19 @@
 <div
-    class='spt-snippet spt-theme-default{% if record.getFormattedDuration() %} spt-has-duration{% endif %}'
+    class='spt-snippet spt-theme-default{% if record.getFormattedDuration() != '0' %} spt-has-duration{% endif %}{% if not record.getResource() or not record.getFormat() %} is-disabled{% endif %}'
```
Comment (votemike) [inline templates/snippet/snippet_record.html.twig]: 0 is an int, not a string.

File: src/ExternalApi/Tupac/Domain/Record.php (hunk 2 above)
Comment (Tasiobg) [reply src/ExternalApi/Tupac/Domain/Record.php]: snippet doesn't support translations, this is same text the snippet project has

File: templates/snippet/snippet_record.html.twig (hunk 5 above)
Comment (Tasiobg) [reply templates/snippet/snippet_record.html.twig]: sorry I forgot to revert this change, thanks

**Pull request merged**